specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/artifact.json

```diff
@@ -0,0 +1,1690 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-05-01",
+    "title": "Artifact API"
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/artifact/v2.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/artifacts/metadata": {
+      "post": {
+        "tags": [
+          "Artifact"
+        ],
+        "summary": "Create Artifact.",
+        "description": "Create an Artifact.",
+        "operationId": "Artifacts_Create",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
```

Comment:
I will look into it

Reply:
@devigned This might not be a simple fix as our services use swashbuckle to autogenerate swaggers. Swashbuckle doesnt seem to support Parameter References.